liblwgeom/liblwgeom.h.in

```diff
@@ -1833,7 +1833,7 @@ extern size_t gserialized_from_lwgeom_size(const LWGEOM *geom);
 * will contain the size of the final output, which is useful for setting the PgSQL 
 * VARSIZE information.
 */
-extern GSERIALIZED* gserialized_from_lwgeom(LWGEOM *geom, int is_geodetic, size_t *size);
+extern GSERIALIZED* gserialized_from_lwgeom(LWGEOM *geom, size_t *size);
```

Comment:
Go ahead do change it, fingers crossed, I'm pretty sure it's an artifact of the 1.5 era when there was no way to flag objects geodetic/not. I'm a little surprised how infrequently that `gserialized_from_lwgeom` shows up in the code, probably that's indicative of it not being the preferred serialization function, and it should be generally changed to something else and then made private. But commit this first.